DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -1,12 +1,69 @@
-﻿using Microsoft.AspNetCore.Mvc;
-
-namespace DigitalLearningSolutions.Web.Controllers
-{
-    public class HomeController : Controller
-    {
-        public IActionResult Index()
-        {
-            return View();
-        }
-    }
-}
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace DigitalLearningSolutions.Web.Controllers
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.ViewModels.Common.MiniHub;
+    using DigitalLearningSolutions.Web.ViewModels.Home;
+    using Microsoft.Extensions.Configuration;
+
+    public class HomeController : Controller
+    {
+        private const string LandingPageMiniHubName = "Digital Learning Solutions";
+        private readonly List<MiniHubSection> sections = new List<MiniHubSection>(
+            new[]
+            {
+                new MiniHubSection(sectionTitle: "Welcome", controllerName: "Home", actionName: "Welcome"),
+                new MiniHubSection(sectionTitle: "Products", controllerName: "Home", actionName: "Products"),
+                new MiniHubSection(
+                    sectionTitle: "Learning Content",
+                    controllerName: "Home",
+                    actionName: "LearningContent"),
+            });
+
+        private readonly IConfiguration configuration;
+
+        public HomeController(IConfiguration configuration)
+        {
+            this.configuration = configuration;
+        }
+
+        public IActionResult Index()
+        {
+            return RedirectToAction("Welcome");
+        }
+
+        [HttpGet]

```

Comment:
Is this attribute necessary?

Reply:
Possibly not, though I find it helps make the code quicker to read / understand, and I don't know of any drawbacks - do you think it's better to remove it.?

Reply:
To me, it makes the code a bit clunkier to read, but it's so marginal that if it helps you, I'm happy for it to stay! :)